include/configs/xenguest_arm64.h

```diff
@@ -15,8 +15,9 @@
 
 #undef CFG_EXTRA_ENV_SETTINGS
 #define CFG_EXTRA_ENV_SETTINGS	\
-	"loadimage=ext4load pvblock 0 0x90000000 /boot/Image;\0" \
-	"pvblockboot=run loadimage;" \
-		"booti 0x90000000 - 0x88000000;\0"
+	"pvblockboot=ext4load pvblock 0 0x90000000 /boot/Image;" \
+		"booti 0x90000000 - 0x88000000;\0" \
+	"virtioboot=virtio scan; ext4load virtio 0 0x90000000 /boot/Image;" \
```

Comment:
Should it be under `#ifdef CONFIG_VIRTIO_BLK` ?

Reply:
Done